projects/storefrontapp-e2e-cypress/cypress.config.ci.ts

```diff
@@ -29,6 +29,7 @@ export default defineConfig({
     OCC_PREFIX: '/occ/v2',
     OCC_PREFIX_USER_ENDPOINT: 'users',
     OCC_PREFIX_ORDER_ENDPOINT: 'orders',
+    RECAPTCHA_PUBLIC_KEY: '6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI',
```

Comment:
For security reasons, I would use a different key exclusive stored as a secret to the CI's domain(s). This is so that not anybody with `localhost` can use our key.